jina/drivers/rank.py

```diff
@@ -64,15 +65,15 @@ def _apply_all(self, docs: Iterable['jina_pb2.Document'], context_doc: 'jina_pb2
         match_idx = []  # type: List[Tuple[int, int, int, float]]
         query_chunk_meta = {}  # type: Dict[int, Dict]
         match_chunk_meta = {}  # type: Dict[int, Dict]
-        for c in docs:
-            for match in c.matches:
+        for doc in docs:
```

Comment:
I wondered, whether `chunk` would be the better naming here. There is one reason, that speaks against it: you could apply this function also against the matches of the `context_doc`. Anyhow, the driver is not appropriate for this task, since the `context_doc` would then have the old matches and the newly calculated matches mixed up. That is bad.

Thus, do we want to rename it `chunk`? (`context_doc_chunk` is to bulky)

Reply:
ok we can rename to `chunk` to make it more explicit that it should be at least someone's chunk